docs/manifest.md

```diff
@@ -78,6 +78,74 @@ examples/foo/bar:
     - if: IDF_TARGET == "esp32s2"
 ```
 
+## Extending and Removing Elements from a YAML List
```

Comment:
```
## Enhanced YAML Syntax

### Reuse Lists

...

#### Array Elements as Strings

...

#### Array Elements as If Clauses
```

I'm moving one section level up for `enhanced yaml syntax`, in case we have other features in the future.